packages/liveblocks-react/src/room.tsx

```diff
@@ -2402,48 +2583,326 @@ export function generateQueryKey(
+ */
+const _useAddReaction: TypedBundle["useAddReaction"] = useAddReaction;
+
+/**
+ * Create a callback function that lets you mutate Liveblocks state.
+ *
+ * The first argument that gets passed into your callback will be
+ * a "mutation context", which exposes the following:
+ *
+ *   - `root` - The mutable Storage root.
+ *              You can normal mutation on Live structures with this, for
+ *              example: root.get('layers').get('layer1').set('fill',
+ *              'red')
+ *
+ *   - `setMyPresence` - Call this with a new (partial) Presence value.
+ *
+ *   - `self` - A read-only version of the latest self, if you need it to
+ *              compute the next state.
+ *
+ *   - `others` - A read-only version of the latest others list, if you
+ *                need it to compute the next state.
+ *
+ * useMutation is like React's useCallback, except that the first argument
+ * that gets passed into your callback will be a "mutation context".
+ *
+ * If you want get access to the immutable root somewhere in your mutation,
+ * you can use `root.ToImmutable()`.
+ *
+ * @example
+ * const fillLayers = useMutation(
+ *   ({ root }, color: Color) => {
+ *     ...
+ *   },
+ *   [],
+ * );
+ *
+ * fillLayers('red');
+ *
+ * const deleteLayers = useMutation(
+ *   ({ root }) => {
+ *     ...
+ *   },
+ *   [],
+ * );
+ *
+ * deleteLayers();
+ */
+const _useMutation: TypedBundle["useMutation"] = useMutation;
+
+/**
+ * Returns a function that creates a thread with an initial comment, and optionally some metadata.
+ *
+ * @example
+ * const createThread = useCreateThread();
+ * createThread({ body: {}, metadata: {} });
+ */
+const _useCreateThread: TypedBundle["useCreateThread"] = useCreateThread;
+
+/**
+ * Returns a function that deletes a thread and its associated comments.
+ * Only the thread creator can delete a thread, it will throw otherwise.
+ *
+ * @example
+ * const deleteThread = useDeleteThread();
+ * deleteThread("th_xxx");
+ */
+const _useDeleteThread: TypedBundle["useDeleteThread"] = useDeleteThread;
+
+/**
+ * Returns a function that edits a thread's metadata.
+ * To delete an existing metadata property, set its value to `null`.
+ *
+ * @example
+ * const editThreadMetadata = useEditThreadMetadata();
+ * editThreadMetadata({ threadId: "th_xxx", metadata: {} })
+ */
+const _useEditThreadMetadata: TypedBundle["useEditThreadMetadata"] =
   useEditThreadMetadata;
-const _useEventListener: DefaultRoomContextBundle["useEventListener"] =
-  useEventListener;
-const _useMyPresence: DefaultRoomContextBundle["useMyPresence"] = useMyPresence;
-const _useOthersMapped: DefaultRoomContextBundle["useOthersMapped"] =
-  useOthersMapped;
-const _useOthersMappedSuspense: DefaultRoomContextBundle["suspense"]["useOthersMapped"] =
+
+/**
+ * useEventListener is a React hook that allows you to respond to events broadcast
+ * by other users in the room.
+ *
+ * @example
+ * useEventListener(({ connectionId, event }) => {
```

Comment:
Should we mention `user` too? 

Also it's `null` when broadcast from server, we could make a note: https://liveblocks.io/docs/api-reference/liveblocks-react#useEventListener

Reply:
Good point, fixed in a94d52e4f0611ea8f87293bd683d132ad520af2e 👍